utils/check_copyright.py

```diff
@@ -129,7 +130,7 @@ def insert_copyright(author, glob, comment_prefix):
         update_file = False
         for line in fileinput.input(file, inplace=1):
             emit = True
-            if state is 0:
+            if state == 0:
```

Comment:
This change has nothing to do with your PR, so don't make it here. If you think "is" is wrong and "==" would be better then feel free to do a separate PR for that, with a PR description explaining the problem that your PR rectifies.

(Similar for the other changes below - you could fix them all in one PR if you think they are really wrong. But this PR isn't the right place to fix them. BTW: I don't know Python that well so I don't know whether "is" or "==" is correct here.)

Reply:
actually, I think we use the `is` Keyword if we want to check if two variables refer to the same object, and use the `==` operator to check if two variables are equal, e.g.

```python 
    x = ['1', '2', '3']
    y = ['1', '2', '3']
    print(x is y) # False 
    print(x == y) # True

```

Reply:
@afd I've back to the original, are you want to make separate PR for this according to explain above?

Reply:
Yes please!

Reply:
done.